packages/rum/src/domain/record/observers/observers.spec.ts

```diff
@@ -1,84 +0,0 @@
-import { isIE, relativeNow, timeStampNow } from '@datadog/browser-core'
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏it seems that after your refacto there are a couple of empty files like this `observers.spec.ts`.
What about adding the lint rule `'unicorn/no-empty-file': 'error'` and removing them?

Reply:
I did remove the empty files but yes having 'unicorn/no-empty-file': 'error' could be nice. I'll add it